changelogs/unreleased/gh-1803-cross-engine-tx.md

```diff
@@ -0,0 +1,7 @@
+## feature/box
+
+* Added support for cross-engine transactions. It is now possible to mix
+  statements for different storage engines in the same transaction. Note
+  that to mix memtx and vinyl statements without receiving `TRANSACTION_YIELD`
```

Comment:
The changelog entry seems to be outdated. The doc request too.

Reply:
Right, removed the mention of the `TRANSACTION_YIELD` error from the change log. As for the doc request, it seems fine: it explicitly requests MVCC for using memtx and vinyl in the same transactions.